cluster/manifests/external-dns/rbac.yaml

```diff
@@ -0,0 +1,42 @@
+{{ if index .ConfigItems "enable_rbac"}}
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: external-dns
+  namespace: kube-system
+  labels:
+    application: external-dns
+---
+# allows to list services and ingresses
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: external-dns
+  namespace: kube-system
```

Comment:
`ClusterRoles` or `ClusterRoleBindings` are not namespaced. I don't know if this makes it break or it just ignores it.

Reply:
That doesn't make sense 😆 Fixed.